cmd/commands.go

```diff
@@ -170,6 +169,9 @@ var gbHelpCmd = &cobra.Command {
 	Args:					cobra.RangeArgs(0, 1),
 }
 
+//gbHelpFunc takes a pointer to cobra.command and
```

Comment:
@kazishafin — There should be a space between `//` and `gbHelpFunc` and for all other similar comments.

Reply:
ok